src/data/content/design-system/componenti/popover.yaml

```diff
@@ -113,7 +211,15 @@ tabs:
             title: Anatomia
             headingLevel: 2
             text: |
-              _In stesura_
+              ![Gli elementi che formano il popover](/images/design-system/componenti/popover-anatomy.png)
+
+              Il componente Buttons è composto da:
```

Comment:
```suggestion
              Il componente Popover è composto da:
```

Reply:
anche in questo caso, minuscolo o maiuscolo?